src/olympia/shelves/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+
+from olympia.shelves.models import Shelf, ShelfManagement
+
+
+class ShelfSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Shelf
+        fields = ['id', 'title', 'endpoint', 'criteria',
```

Comment:
frontend won't need `id`.

Reply:
And `endpoint` and `criteria` won't mean much either - they should be combined into a url field.  (url can be a `SerializerMethodField` that returns a full api url - similar to how the url is being built in the clean methods)